### PR TITLE
Update views immediately when changing verified field in admin bro

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to https://github.com/Giveth/giveth-dapps-v2/issues/1736#issuecomment-1314968790

### CODE REVIEW

File: src/server/adminBro.ts [hunk 1]
```diff
@@ -2287,6 +2292,10 @@ export const verifyProjects = async (
         }
       }
     }
+
+    await refreshUserProjectPowerView();
+    await refreshProjectPowerView();
+    await refreshProjectFuturePowerView();
```
Comment (aminlatifi) [inline src/server/adminBro.ts]: I would do it in one line
```
await Promise.all([
    refreshUserProjectPowerView(),
    refreshProjectPowerView(),
    refreshProjectFuturePowerView()
])
```

**Review by aminlatifi — CHANGES_REQUESTED** [review]
A project can be unverified in `src/services/cronJobs/checkProjectVerificationStatus.ts` too. We must refresh the views there if anything changes

File: src/server/adminBro.ts (hunk 1 above)
Comment (mohammadranjbarz) [reply src/server/adminBro.ts]: Thanks @aminlatifi , you are absolutely right, I did it.

**Comment by aminlatifi** [comment]
> A project can be unverified in `src/services/cronJobs/checkProjectVerificationStatus.ts` too. We must refresh the views there if anything changes

This one has remained

**Comment by mohammadranjbarz** [comment]
> > A project can be unverified in `src/services/cronJobs/checkProjectVerificationStatus.ts` too. We must refresh the views there if anything changes
> 
> This one has remained

I misunderstood before, now it's done

File: src/services/cronJobs/checkProjectVerificationStatus.ts [hunk 2]
```diff
@@ -173,6 +169,11 @@ const remindUpdatesOrRevokeVerification = async (project: Project) => {
   const user = await User.findOne({ id: Number(project.admin) });
 
   await sendProperNotification(project, project.verificationStatus as string);
+  await Promise.all([
```
Comment (aminlatifi) [inline src/services/cronJobs/checkProjectVerificationStatus.ts]: I think we must to it in `checkProjectVerificationStatus` in when  the list of projects to unverify are not empty.

**Review by aminlatifi — CHANGES_REQUESTED** [review]

**Review by aminlatifi — APPROVED** [review]

**Pull request merged**